Description: change path from opt to usr/share
 change path from opt to usr/share
Author: Andreas Wirooks <andreas.wirooks@1und1.de>

--- a/pkg/logstash-web.default
+++ b/pkg/logstash-web.default
@@ -18,7 +18,7 @@
 LS_USER=logstash
 
 # location of the logstas jar file
-LS_JAR=/opt/logstash/logstash.jar
+LS_JAR=/usr/share/logstash/logstash.jar
 
 # logstash home location
 LS_HOME=/var/lib/logstash
--- a/pkg/logstash.default
+++ b/pkg/logstash.default
@@ -18,7 +18,7 @@
 LS_USER=logstash
 
 # location of the logstas jar file
-LS_JAR=/opt/logstash/logstash.jar
+LS_JAR=/usr/share/logstash/logstash.jar
 
 # logstash home location
 LS_HOME=/var/lib/logstash
